composer.json

```diff
@@ -6,19 +6,29 @@
         {
             "name": "Wouter Peschier",
             "email": "wouter@pixelfusion.co.nz"
+        },
```

Comment:
we should have a requirement for the version of php we support, we seem to be using type hints that only available in 7+